src/proto/grpc/testing/test.proto

```diff
@@ -47,6 +47,11 @@ service TestService {
   // One request followed by one response.
   rpc UnaryCall(SimpleRequest) returns (SimpleResponse);
 
+  // One request followed by one response. Response has cache control
+  // headers set such that a caching HTTP proxy (such as GFE) can
+  // satisfy subsequent requests.
+  rpc CacheableUnaryCall(SimpleRequest) returns (SimpleResponse);
```

Comment:
It doesn't have any option on it to define it as safe/cachable. Since that is missing, manually specifying in the test seems required. We should specify that, although we should probably note that it is temporary until this proto is fixed.